tests/unit/config/test_env_var.py

```diff
@@ -31,6 +32,13 @@ def test_value_bad(monkeypatch, caplog, empty_conf):
     assert "invalid literal" in caplog.messages[0]
 
 
+def test_python_via_env_var(monkeypatch):
+    options = VirtualEnvOptions()
+    monkeypatch.setenv(str("VIRTUALENV_PYTHON"), str("python3"))
```

Comment:
Can you add a test to validate that ``python2,python3`` works as expected too?

Reply:
I extended the code to provide that functionality. Multi-value was split with newline so far, though it seems the only other multi-value field is `extra-search-dir`.

Let me know if you are happy with the approach taken or would prefer a different way to do this.